bindings/python/capstone/__init__.py

```diff
@@ -195,8 +199,10 @@
 CS_MODE_M680X_6811 = (1 << 8)  # M680X M68HC11 mode
 CS_MODE_M680X_CPU12 = (1 << 9)  # M680X CPU12 mode
 CS_MODE_M680X_HCS08 = (1 << 10)  # M680X HCS08 mode
-CS_MODE_BPF_CLASSIC = 0        # Classic BPF mode (default)
-CS_MODE_BPF_EXTENDED = 1 << 0  # Extended BPF mode
+CS_MODE_BPF_CLASSIC = 0          # Classic BPF mode (default)
+CS_MODE_BPF_EXTENDED = (1 << 0)  # Extended BPF mode
+CS_MODE_RISCV32 = (1 << 0)       # RISCV32 mode
+CS_MODE_RISCV64 = (1 << 1)       # RISCV64 mode
```

Comment:
how about `CS_MODE_RISCVC`?

Reply:
RISCVC especially has not any code added for capstone, for now, So, add it or not is meaningless.

Reply:
but since you have RISCVC in `riscv.h`, it should be here too, for consistency.